src/jit/liveness.cpp

```diff
@@ -2050,7 +2053,10 @@ void Compiler::fgComputeLifeLIR(VARSET_TP& life, BasicBlock* block, VARSET_VALAR
                     DISPNODE(lclVarNode);
 
                     blockRange.Delete(this, block, node);
-                    fgStmtRemoved = varDsc.lvTracked && !opts.MinOpts();
+                    if (varDsc.lvTracked && !opts.MinOpts())
```

Comment:
Why do we have `!opts.MinOpts()` condition here? Why is it correct not to recompute liveness in min-opts when we delete lclVarUse that was last use?

Reply:
I'm not sure; from a register allocation perspective, we never put lclVars in registers under MinOpts, and I *thought* it was the case that we didn't even have tracked lclVars, but perhaps I am mistaken about that.